api/src/opentrons/protocol_engine/state/__init__.py

```diff
@@ -4,11 +4,19 @@
 from .commands import CommandState, CommandView, CommandSlice, CurrentCommand
 from .labware import LabwareState, LabwareView
 from .pipettes import PipetteState, PipetteView, HardwarePipette, CurrentWell
-from .modules import ModuleState, ModuleView, MagneticModuleView, HardwareModule
+from .modules import ModuleState, ModuleView, HardwareModule
+from .module_substates import (
+    MagneticModuleId,
+    MagneticModuleSubState,
+    HeaterShakerModuleSubState,
+    HeaterShakerModuleId,
+    ModuleViewTypes,
+)
```

Comment:
Nitpick for consistent ordering:

```suggestion
from .module_substates import (
    MagneticModuleId,
    MagneticModuleSubState,
    HeaterShakerModuleId,
    HeaterShakerModuleSubState,
    ModuleViewTypes,
)
```